plasmapy/diagnostics/tests/test_proton_radiography.py

```diff
@@ -365,13 +365,15 @@ def test_run_options():
     # Test extreme deflections -> warns user
     # This requires instatiating a whole new example field with a really
     # big B-field
-    grid = _test_grid("constant_bz", num=50, B0=500 * u.T)
+    grid = _test_grid("constant_bz", num=50, B0=250 * u.T)
     source = (0 * u.mm, -10 * u.mm, 0 * u.mm)
     detector = (0 * u.mm, 200 * u.mm, 0 * u.mm)
     sim = prad.SyntheticProtonRadiograph(grid, source, detector, verbose=False)
-    sim.create_particles(1e4, 3 * u.MeV, max_theta=1 * u.deg)
+    sim.create_particles(1e4, 3 * u.MeV, max_theta=10 * u.deg)
     with pytest.warns(RuntimeWarning):
         sim.run(field_weighting="nearest neighbor", dt=1e-12 * u.s)
+        # Calc max deflection
+        sim.max_deflection
```

Comment:
Would be good to at least have a bounds check (`assert lower < sim.max_deflection < upper`) here, rather than calculating it just for the non-exception-throwing-execution code coverage!

Reply:
Good idea - I've set a bound between 0 and pi/2 (which is valid for particles with a small `max_theta`).